frontend/src/lib/stores.js

```diff
@@ -1,20 +1,38 @@
-import words from '../languages/et/words.json'
 import { writable } from 'svelte/store'
+import languages from '../languages/languages.json'
+
+export const currentLanguage = writable(languages[0])
+let _currentLanguage
+currentLanguage.subscribe((value) => (_currentLanguage = value))
+
+async function loadWords(code) {
+  return (await import(`../languages/${code}/words.json`)).default
+}
 
 function randInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min
 }
 
-function getRandomWord() {
+async function getRandomWord() {
+  let words = await loadWords(_currentLanguage.code)
```

Comment:
`loadWords` should only be called when the language changes, i.e. in `currentLanguage.subscribe()`

Reply:
Or rather, the `import` should only be called when the language has changed, so you could also implement the caching for that in that function

Reply:
if `loadWords` should only be called when the language changes, how do we initialize `words `?